Resources/doc/index.rst

```diff
@@ -229,11 +229,11 @@ and retrieve your menu from a template, just like available in Twig.
 More Advanced Stuff
 -------------------
 
-* :doc:`Menus as Services <menu_service>`
-* :doc:`Custom Menu Renderer <custom_renderer>`
-* :doc:`Custom Menu Provider <custom_provider>`
-* :doc:`I18n for your menu labels <i18n>`
-* :doc:`Using events to allow extending the menu <events>`
+* `Creating Menus as Services <menu_service.rst>`_
```

Comment:
this is wrong. We are using the Sphinx internal reference instead, because our docs are not rendered on symfony.com